src/eq_schema/RoutingRule.test.js

```diff
@@ -234,35 +238,39 @@ describe("Rule", () => {
     });
   });
 
-  it("should build valid runner routing to next page if its the last page in questionnaire", () => {
-    const lastPage = createRuleJSON(nextPageGoto, basicRadioCondition);
+  // eslint-disable-next-line jest/no-disabled-tests
+  xit(
```

Comment:
I guess it could be deleted. Depends if disabling routing on the last page of a section is a permanent change or something that we would re-enable once routing between sections is supported.

The test was disabled because it no longer applies (since we don't process routing rules for the last page in a section). But that's not to say we may re-introduce it at some point.